components/mdc/IconButton/IconButton.svelte

```diff
@@ -25,15 +26,17 @@ onMount(() => {
     class="mdc-icon-button material-icons {$$props.class}"
     aria-label={ariaLabel}
     {disabled}
+    {target}
```

Comment:
In the story, the target value is `"_blank"`. With this implementation, wouldn't it have to be `target="_blank"`?

Reply:
We don't want a new tab opening by default as that would be a breaking change and isn't how we've used these components in the past. target can be a few different values like "_self". I still need to make sure having target='' behaves as expected in our apps.

Reply:
Well, my point was that you don't include the `target=` part anywhere. It is not in the story and not in the component. If you put it in the component, it's a breaking change, right?

Reply:
It is getting passed in to the `<a {target}` syntax equivalent to `<a target={target}` . Maybe I don't understand what you're saying?

Reply:
Oh, I see. Sorry about my bad js parsing.